frontend/cinema-ticket-booking/src/Endpoints.js

```diff
@@ -0,0 +1,9 @@
+const mainEndpoint = "http://localhost:8080/";
+
+const getAllMovies = mainEndpoint + "movies";
+const getSingleMovie = (id) => {
+    return mainEndpoint + "movies/" + id;
+}
+const putMovie = mainEndpoint + "movies";
```

Comment:
Jest jakaś różnica pomiędzy "getAllMovies", a "putAllMovies" poza nazwą zmiennej, czy w sumie można byłoby to nazwać AllMovies i w jakiejś funkcji potem wybiera się, czy GET czy PUT?

Reply:
Nie ma różnicy, ale w przyszłości być może będziemy rozróżniać metody GET od PUT więc warto już takie rozróżnienie zrobić teraz we frontendzie